messaging/messaging-core/src/main/java/uk/gov/justice/services/messaging/JsonObjects.java

```diff
@@ -281,5 +283,17 @@ private static void checkArguments(final JsonObject object, final String... name
         }
     }
 
-
+    /**
+     * Convert a collection of objects into a JsonArray
+     *
+     * @param entries the collection to be converted
+     * @param converter to convert each entry in the collection
+     * @param <T> the type of element in the collection
+     * @return a JsonArray with the converted entries
+     */
+    public static <T> JsonArray toJsonArray(Collection<T> entries, Function<T,JsonValue> converter) {
+        JsonArrayBuilder arrayBuilder = Json.createArrayBuilder();
```

Comment:
Jason.createArrayBuilder could be a static import.
entries and coverer params to be marked as final
arrayBuilder to be marked as final
Formatting, add space between <T, JsonValue>